debian/changelog

```diff
@@ -1,3 +1,18 @@
+opx-nas-acl (2.8.0) unstable; urgency=medium
+
+  * Update: Increase service timeout
+  * Update: Put set_user_trap_id action to end of updating list to avoid validation check fail
+  * Update: Add priority value to argument of NDI UDF match create function and fixed default L3 type mask value error
```

Comment:
"fixed default L3 type mask value error" -- this should be called out as a "bugfix" right?